internals/src/Tizen.Peripheral/Tizen.Peripheral/Uart.cs

```diff
@@ -0,0 +1,357 @@
+﻿/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeUart = Interop.Peripheral.Uart;
+
+namespace Tizen.Peripheral.Uart
+{
+    /// <summary>
+    /// Enumeration for baud rate.
+    /// </summary>
+    public enum BaudRate
+    {
+        /// <summary>The number of signal in one second is 0.</summary>
+        Rate0 = 0,
+
+        /// <summary>The number of signal in one second is 50.</summary>
+        Rate50,
+
+        /// <summary>The number of signal in one second is 75.</summary>
+        Rate75,
+
+        /// <summary>The number of signal in one second is 110.</summary>
+        Rate110,
+
+        /// <summary>The number of signal in one second is 134.</summary>
+        Rate134,
+
+        /// <summary>The number of signal in one second is 150.</summary>
+        Rate150,
+
+        /// <summary>The number of signal in one second is 200.</summary>
+        Rate200,
+
+        /// <summary>The number of signal in one second is 300.</summary>
+        Rate300,
+
+        /// <summary>The number of signal in one second is 600.</summary>
+        Rate600,
+
+        /// <summary>The number of signal in one second is 1200.</summary>
+        Rate1200,
+
+        /// <summary>The number of signal in one second is 1800.</summary>
+        Rate1800,
+
+        /// <summary>The number of signal in one second is 2400.</summary>
+        Rate2400,
+
+        /// <summary>The number of signal in one second is 4800.</summary>
+        Rate4800,
+
+        /// <summary>The number of signal in one second is 9600.</summary>
+        Rate9600,
+
+        /// <summary>The number of signal in one second is 19200.</summary>
+        Rate19200,
+
+        /// <summary>The number of signal in one second is 38400.</summary>
+        Rate38400,
+
+        /// <summary>The number of signal in one second is 57600.</summary>
+        Rate57600,
+
+        /// <summary>The number of signal in one second is 115200.</summary>
+        Rate115200,
+
+        /// <summary>The number of signal in one second is 230400.</summary>
+        Rate230400
+    }
+
+    /// <summary>
+    /// Enumeration for byte size.
+    /// </summary>
+    public enum ByteSize
```

Comment:
```suggestion
    public enum DataBits
```
https://docs.microsoft.com/ko-kr/dotnet/api/system.io.ports.serialport.databits?view=dotnet-plat-ext-3.1

Reply:
It means length of data bits per byte.